### DISCUSSION TITLE
How to move cursor between panels?

### DISCUSSION THREAD

**Discussion by Hecatoncheir** [JetBrains/ideavim#800] [Q&A]
In [lazyvim](https://github.com/LazyVim/LazyVim/blob/879e29504d43e9f178d967ecc34d482f902e5a91/lua/lazyvim/config/keymaps.lua#L14) it works like that: 

https://github.com/JetBrains/ideavim/assets/910917/7db6dc8d-878b-468e-89a6-059c29c39875

I try add to .ideavimrc:
```
map <C-h> <C-W>h
map <C-l> <C-W>l
map <C-k> <C-W>k
map <C-j> <C-W>j
```
but it does not move to project tree or file structure.

Is it possible to do this?

**Comment by gcalica** [comment]
Unfortunately it is not possible. This is a limitation of the plugin as it is only an editor extension, so it doesn't understand context outside of the editor and thus moving between splits to get to the project structure wont work.

You can technically still move from the editor tab to the project tree (or any other tool windows) with IdeaVim by mapping the other actions (i.e., ActivateProjectToolWindow), but you cannot jump back again to the editor with IdeaVim mappings directly because by then you are outside of editor context.

**Comment by lippfi** [comment]
I understand that it's not enough to cover all the cases, but we have a NERDTree plugin, which may be helpful - https://github.com/JetBrains/ideavim/wiki/IdeaVim-Plugins#nerdtree
